### PR TITLE
open-cluster-management/backlog#2472 Pull secret

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

**Comment by cadawson** [comment]
@TheRealHaoLiu and @nathanweatherly : Can you take a look at the updates to these topics?  This is intended to cover the need to create a pull secret only when you are going to import a non-OCP cluster, and the replacement of quay.io with registry.redhat.io. @TheRealHaoLiu : The files in the acm-hive-openshift-releases repo still contains references to quay.io.  Should I leave it as quay.io in the cluster image sets content?  Thanks!

**Comment by cadawson** [comment]
@TheRealHaoLiu and @nathanweatherly : Can you take a look at the updates to these topics? This is intended to cover the need to create a pull secret only when you are going to import a non-OCP cluster, and the replacement of quay.io with registry.redhat.io. Thanks!

**Comment by nathanweatherly** [comment]
Sorry for the delay @cadawson . Install section looks good! Thanks a bunch!

File: install/install_connected.md [hunk 1]
```diff
@@ -85,10 +85,10 @@ You must meet the following requirements before you install Red Hat Advanced Clu
...
+5. If you plan to import Kuberntetes clusters that were not created by OpenShift Container Platform or Red Hat Advanced Cluster Management, generate a pull secret to access the entitled content from the distribution registry. Pull secret requirements for OpenShift Container Platform clusters are automatically resolved by OpenShift Container Platform and Red Hat Advanced Cluster Management. You do not have to generate the pull secret if you are not importing other types of Kubernetes clusters to be managed. **Important:** Pull secrets are namespace-specific, so make sure that you are in the namespace that you created in step 1.
   
    ```
-   oc create secret docker-registry <secret> --docker-server=registry.access.redhat.com/rhacm1-tech-preview --docker-username=<docker_username> --docker-password=<docker_password>
+   oc create secret docker-registry <secret> --docker-server=registry.redhat.io/rhacm1-tech-preview --docker-username=<docker_username> --docker-password=<docker_password>
```
Comment (TheRealHaoLiu) [inline install/install_connected.md:91]: i dont think this will work
@chenz4027 can u provide the instruction that we use

Comment (chenz4027) [reply install/install_connected.md:91]: the openshift one?

Comment (chenz4027) [reply install/install_connected.md:91]: https://github.com/open-cluster-management/backlog/issues/2596

**Comment by cadawson** [comment]
Hi folks.  I updated the pull secret steps. Can you take a look again? @nathanweatherly I wanted to make sure that you also saw the new changes.  @chenz4027 @TheRealHaoLiu @dhaiducek . Thanks!

File: install/install_connected.md (hunk 1 above)
Comment (cadawson) [reply install/install_connected.md:93]: @chenz4027 Do you know if there is a command that will work for the CLI, if this one won't?

Comment (dhaiducek) [reply install/install_connected.md:93]: The CLI possibility I've found is to download the Openshift pull-secret and then run
```
oc create secret generic <secret-name> -n <namespace> --from-file=<path/to/pull-secret>
```
I haven't tested it, though.

Comment (dhaiducek) [reply install/install_connected.md:93]: ^^^ (Updated)

Comment (chenz4027) [reply install/install_connected.md:93]: I'm not too sure about cli, but I got mine from https://cloud.redhat.com/openshift/install/aws/installer-provisioned -> click copy pull secre
and paste the info in the big square 
image

Comment (clyang82) [reply install/install_connected.md:93]: I have tested the CLI from @dhaiducek update, that works. Thanks @dhaiducek . In APAC Testathon, all people met the issue that cannot pull the image issue during import EKS cluster. Thanks @chenz4027 for your PR.

File: manage_cluster/import_cli.md [hunk 2]
```diff
@@ -42,12 +42,14 @@ After you install Red Hat Advanced Cluster Management for Kubernetes, you are re
    oc new-project ${CLUSTER_NAME}
    ```
 
-3. Run the following command to create a secret:
+3. If you are importing a Kubernetes cluster that is not a Red Hat OpenShift Container Platform cluster, run the following command to create a secret:
 
    ```
-   oc create -n ${CLUSTER_NAME} secret docker-registry quay-secret --docker-server=quay.io --docker-username=${DOCKER_USER} --docker-password=${DOCKER_PASS}
+   oc create -n ${CLUSTER_NAME} secret docker-registry quay-secret --docker-server=registry.redhat.io --docker-username=${DOCKER_USER} --docker-password=${DOCKER_PASS}
```
[the remark this answers is not in the document]
Comment (cadawson) [reply manage_cluster/import_cli.md:48]: Thanks @dhaiducek !  I agree that it is better to use the generic link. I didn't see a way to get to that one, so I mentioned that they were all the same to cover it.  I have replaced that link.

**Comment by cadawson** [comment]
It looks like this is better than what we had before, and Nathan approved, so I am going to merge.  We can make some tweaks, if we need to, but I would rather get an improvement out there than wait. Thanks!

**Pull request merged**